backend/src/main/java/org/atlantbh/internship/auctionapp/service/api/StripeService.java

```diff
@@ -0,0 +1,19 @@
+package org.atlantbh.internship.auctionapp.service.api;
+
+import com.stripe.exception.StripeException;
+import com.stripe.model.Customer;
+import com.stripe.model.PaymentMethod;
```

Comment:
This is a payment interface that should not depend on Stripe. Stripe is just the current implementation. Use custom classes and exceptions instead.